zqd/handlers.go

```diff
@@ -65,11 +65,11 @@ func handlePacketSearch(c *Core, w http.ResponseWriter, r *http.Request) {
 	if err := req.FromQuery(r.URL.Query()); err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 	}
-	if s.PacketPath() == "" || !s.HasFile(packet.IndexFile) {
+	if s.PacketPath() == "" || !s.HasFile(ingest.IndexFile) {
```

Comment:
Would you rename `IndexFile` in the new `ingest` package to `PcapIndexFile` (or similar)?

Reply:
Done